src/Core/Content/Seo/SalesChannel/SeoUrlRouteResponse.php

```diff
@@ -12,10 +12,8 @@ class SeoUrlRouteResponse extends StoreApiResponse
 {
     /**
      * @var EntitySearchResult<SeoUrlCollection>
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
      */
-    protected $object;
+    protected EntitySearchResult $object;
```

Comment:
please do not change the StoreApiResponse, as it will be done be me for all here: https://github.com/shopware/shopware/pull/6555